### DISCUSSION TITLE
I want to always fix the tone of ChatGPT.

### DISCUSSION THREAD

**Discussion by Kitamog** [mishl-dev/Discord-AI-Chatbot#42] [Q&A]
You can fix the tone by asking ChatGPT to speak in the tone of "XX". But it is troublesome to ask every time. Is there any way to set it automatically in advance?

**Comment by mishl-dev** [comment]
> You can fix the tone by asking ChatGPT to speak in the tone of "XX". But it is troublesome to ask every time. Is there any way to set it automatically in advance?

look at like 35 and you can now use /slash commands

![image](https://user-images.githubusercontent.com/91066601/236654730-b172a6ce-e770-4df2-968b-359f81d6d97a.png)
